libs/core/algorithms/include/hpx/parallel/container_algorithms/reverse.hpp

```diff
@@ -232,7 +232,7 @@ namespace hpx { namespace ranges {
     ///
     /// \returns  The \a reverse_copy algorithm returns a
     ///           \a ranges::reverse_copy_result<
-    ///           typename hpx::traits::range_iterator<Rng>::type, OutIter>>::type.
+    ///           typename hpx::traits::range_iterator<Rng>::type, OutIter>.
```

Comment:
The `>::type` should not go away, I rather think that the two line above should be combined into one as doxygen is confused by the line split, not the `::type`